src/shared/components/event/event.component.html

```diff
@@ -1,43 +1,44 @@
 <div class="event-wrapper">
-<div class="event-container"  >
-  <div class="image-wrapper"[routerLink]="event._id">
-    <img
-      alt="{{ event.shortTitle }}"
-      src="{{ event.imageUrl }}"
-      class="event-image"
-    />
-  </div>
-  <div class="event-info">
-    <h3 class="event-title">{{ event.shortTitle }}</h3>
-    <p class="event-location">
-      <ion-icon [src]="location" name="location" class="location-img"/>
-      {{ event.contacts.region }}
-    </p>
-    <p *ngIf="event.dates.length > 1" class="event-date">
-      <ion-icon [src]="date" name="date" class="date-img"/>
-      От {{ event.dates[0].date | date : "dd.MM.yyyy" }} г. до
-      {{ event.dates[event.dates.length - 1].date | date : "dd.MM.yyyy" }}
-      г.
-    </p>
-    <p *ngIf="event.dates.length === 1" class="event-date">
-      <ion-icon [src]="date" name="date" class="date-img"/>
-      {{ event.dates[0].date | date : "dd.MM.yyyy" }} г.
-    </p>
-    <p class="event-description">{{ event.shortDescription }}</p>
-    <button class="more-btn" [routerLink]="event._id">
-      <p class="btn-text">Прочети повече</p>
-    </button>
-    
-    <div *ngIf="user?._id == event.creator" class="organizer-btn">
-      <button class="del btn" (click)="deleteEvent(event)"> <!-- Delete have issue with backend -->
-        <p class="del-btn-text">Изтрий</p>
-      </button>
-      <button class="edit btn"> <!-- TODO Link to edit -->
-        <p class="edit-btn-text">Редактирай</p>
+  <div class="event-container" >
+    <div class="image-wrapper"[routerLink]="event._id">
+      <img
+        alt="{{ event.shortTitle }}"
+        [src]="'data:image;base64,' + event.imageUrl"
+        class="event-image"
+      />
+    </div>
+    <div class="event-info">
+      <h3 class="event-title">{{ event.shortTitle }}</h3>
+      <p class="event-location">
+        <ion-icon [src]="location" name="location" class="location-img" />
+        {{ event.contacts.region }}
+      </p>
+      <p *ngIf="event.dates.length > 1" class="event-date">
+        <ion-icon [src]="date" name="date" class="date-img" />
+        От {{ event.dates[0].date | date : "dd.MM.yyyy" }} г. до
+        {{
+          event.dates[event.dates.length - 1].date | date : "dd.MM.yyyy"
+        }}
+        г.
+      </p>
+      <p *ngIf="event.dates.length === 1" class="event-date">
+        <ion-icon [src]="date" name="date" class="date-img" />
+        {{ event.dates[0].date | date : "dd.MM.yyyy" }} г.
+      </p>
+      <p class="event-description">{{ event.shortDescription }}</p>
+      <button class="more-btn" [routerLink]="event._id">
+        <p class="btn-text">Прочети повече</p>
       </button>
+      <div *ngIf="user?._id == event?.creator" class="organizer-btn">
+        <button class="del btn" (click)="OnDelete(event)">
+          <p class="del-btn-text">Изтрий</p>
+        </button>
+        <button class="edit btn"> <!-- TODO Link to edit -->
```

Comment:
and TODO ngIf creator and admin

Reply:
where? The btns can be seen only from creator and admin. Already made it and test.